shared/metrics/src/main/java/io/pravega/shared/MetricsNames.java

```diff
@@ -25,6 +25,12 @@
     public static final String HDFS_READ_BYTES = "tier2_read_bytes";          // Counter
     public static final String HDFS_WRITE_BYTES = "tier2_write_bytes";        // Counter
 
+    //filesystem stats
+    public static final String FILESYSTEM_READ_LATENCY = "tier2_read_latency_ms";   // Timer
```

Comment:
These changes are because of @Abhijeet-Jadhav suggestion here: https://github.com/pravega/pravega/pull/1535#discussion_r125675979. Will rename `HDFS_*` above to `TIER2_*` and update.